phpunit/functional/Glpi/Dashboard/DashboardTest.php

```diff
@@ -159,6 +159,26 @@ public function testClone()
         }
     }
 
+    public function testCloneKeyUnicity()
```

Comment:
I'm not sure the test is really relevant, since you create a brand new dashboard on each iteration.The main dashboard key must also be added to tests checks.